src/test/java/com/arpnetworking/kairosdb/KairosHelper.java

```diff
@@ -147,6 +148,53 @@ public static HttpPost queryFor(
         return lookup;
     }
 
+    /**
+     * Creates the JSON body for a datapoint query.
+     *
+     * @param startTimestamp starting timestamp
+     * @param endTimestamp ending timestamp
+     * @param metric metric to query for
+     * @param aggregators the list of aggregators to be applied in order for the query
+     * @return JSON Object to POST to KairosDB
+     * @throws JSONException on JSON error
+     */
```

Comment:
Awesome. Can we replace the above implementation of `queryFor` with a simple wrapper around this more-general function?